plugins/techdocs/src/alpha.tsx

```diff
@@ -44,8 +68,89 @@ export const TechDocsSearchResultListItemExtension =
     },
   });
 
+/**
+ * Responsible for rendering the provided router element
+ *
+ * @alpha
+ */
+const TechDocsIndexPage = createPageExtension({
```

Comment:
Are there any properties that could be mapped to config schemas? My suggestion would be to start with the `initialFilter`, similar to what we are experimenting with in the `Catalog` plugin:
https://github.com/backstage/backstage/blob/18711f65c2f92a153273af3bcb9de4846f25abdb/plugins/catalog/src/alpha.tsx#L130-L143

Reply:
I'm skipping this for now as `configSchema` for `createPageExtension` requires a `path`. Not sure why it is required & doesn't go with the default, but it would require a config change so I will skip it for now.